### ISSUE TITLE
"Add Asset" Form Displays on Closed Work Order

### ISSUE THREAD

**Issue by johnclary** [cityofaustin/atd-data-tech#442]
I'm looking at the location details on [this](https://atd.knack.com/signs-markings#work-order-signs/view-work-orders-details-sign/5d3f0113674b9e001208986e/view-work-order-signs-location-details/5d433682a954d60011cc2376/) closed work order. The add asset form is exposed when logged in as test-technician. 

It looks like we don't have any page rules in place to hide this form based on status:
https://builder.knack.com/atd/signs-markings#pages/scene_1039/rules

Should also check the "Edit Location" page.

**Comment by dianamartin** [comment]
@johnclary Yeah, something weird happened. I noticed it on Alan's computer too yesterday. I'll check it out.

**Comment by dianamartin** [comment]
- [x] Fixed Page Rules for GIS QA
- [ ] Checking Page Rules for Test Technician
